dashboard/lib/public/javascripts/dashboard.js

```diff
@@ -164,22 +164,21 @@ function loadReport(filename) {
 }
 
 
+function asyncLogout() {
+    setTimeout(function() {
+        Fiware.signOut('health')
```

Comment:
Fiware.signOut is a function defined in layout.jade file in line 13 https://github.com/telefonicaid/fiware-health/pull/221#discussion-diff-50842536R13